src/apps/front-office/design-system/components/topSelling/ProductBox.tsx

```diff
@@ -0,0 +1,61 @@
+import { fullRate, halfRate } from "./topSelling";
+import PropTypes from 'prop-types';
+const ProductBox = ({
```

Comment:
Please use more appropriate and readable component names and props. For example:

- ProductCard instead of ProductBox.
- productImage instead of image.
- productName instead of text.
- priceNow instead of price.
- salePrice instead of priceDelete .

These adjustments will improve clarity and make the component easier to understand.